README.md

```diff
@@ -622,6 +622,54 @@ const storage = createStorage({
 - `storage`: The name of the table to read from. It defaults to `storage`.
 - `boostCache`: Whether to enable cached queries: see [docs](https://planetscale.com/docs/concepts/query-caching-with-planetscale-boost#using-cached-queries-in-your-application).
 
+### `azure-storage-table`
```

Comment:
Can you please check if this works with CF workers and browsers? (we might add (Node) if is node-js supported)

Reply:
It depends on which authentication method is used, this is documented at the specific authentication method in the README and in the typedocs for the parameters. Only `DefaultAzureCredential` and `AzureSASCredential` is supported in browser.

Reply:
CF workers is something I have to test, haven't used the so far, just local workers.

Reply:
@pi0 I would like to merge and validate CF functionality for all of the Azure libraries in the following days when I have finished getting comfortable with CF.

Reply:
The Azure storage SDKs don't seem to be compatible with CF workers currently, especially when using @azure/identity to login. I added a note for that to the README.